src/web/components/Container/Container.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Container from './Container';
+
+describe('Render Container', () => {
+  it('should render children component', () => {
+    const content = <div id="test">toto</div>;
+    const wrapper = shallow(<Container>{content}</Container>);
+    const children = wrapper.find('div#test');
+    expect(children).toHaveLength(1);
```

Comment:
We prefer to use only one assertion / tests 😄 

It's better to split it in multiple cases.

For example, your test `it` says that it should render a component. Not that its children should be 1.

But the test is good actually, just split it into two tests 😄